docs/user_guide/5_changelog.md

```diff
@@ -3,6 +3,38 @@
 
 ## ROS Noetic
 
+### Version 1.0.4
+
+* Xacros refactored
+* removing deprecated sr hand package
```

Comment:
Can you please change all to start with a capital letter?

Reply:
Done